### PR TITLE
samples: add CcRuntime for enclave-cc

### PR DESCRIPTION
**Pull request by mythi**
Signed-off-by: Mikko Ylinen <mikko.ylinen@intel.com>

### CODE REVIEW

File: config/samples/ccruntime-enclave-cc.yaml [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+          type: ""
+        name: systemd
+      - hostPath:
+          path: /usr/local/bin/
+          type: ""
+        name: local-bin
+    installCmd: ["/opt/enclave-cc-artifacts/scripts/enclave-cc-deploy.sh", "install"]
+    uninstallCmd: ["/opt/enclave-cc-artifacts/scripts/enclave-cc-deploy.sh", "cleanup"]
+    cleanupCmd: ["/opt/enclave-cc-artifacts/scripts/enclave-cc-deploy.sh", "reset"]
+    # Uncomment and add the required RuntimeClassNames to be created
```
Comment (fidencio) [inline config/samples/ccruntime-enclave-cc.yaml:59]: Please, remove this comment.

**Review by fidencio — COMMENTED** [review]
@mythi, one little nit, but it looks good!
Thanks for the contribution!

File: config/samples/ccruntime-enclave-cc.yaml (hunk 1 above)
Comment (mythi) [reply config/samples/ccruntime-enclave-cc.yaml]: dropped.

**Review by fidencio — APPROVED** [review]
lgtm, thanks @mythi!

**Review by bpradipt — APPROVED** [review]
/lgtm
Thanks @mythi

**Pull request merged**